src/en/charms-fan.md

```diff
@@ -0,0 +1,207 @@
+Title: Juju and FAN networking
+TODO:  bug tracking: https://bugs.launchpad.net/juju/+bug/1733354 (remove constraints Note if no longer required)
+       hardcoded: default AWS instance type of 'm3.medium'
+       consider a third example: manually configure the FAN with Azure
+
+# Juju and FAN networking
+
```

Comment:
This could maybe use a few words of what we mean by 'FAN'. Is it an acronym? Is it, indeed, FAN and not Fan, as it appears elsewhere

Reply:
I'll change to Fan.